Added check for avi SDK version as suggested in review.

grastogi23 opened the pull request:
##### ISSUE TYPE
<!--- Pick one below and delete the rest: -->
 - Bugfix Pull Request

##### COMPONENT NAME
<!--- Name of the module/plugin/module/task -->
network.avi

##### ANSIBLE VERSION
<!--- Paste verbatim output from “ansible --version” between quotes below -->
```
ansible 2.3.0
  config file = /home/grastogi/.ansible.cfg
  configured module search path = Default w/o overrides
```

##### SUMMARY
<!--- Describe the change, including rationale and design decisions -->
Ansible modules have tight dependency on the avisdk. Added version check for avisdk to inform user of the proper versions they need to use for avisdk.
<!---
If you are fixing an existing issue, please include "Fixes #nnn" in your
commit message and your description; but you should still explain what
the change does.
-->

<!-- Paste verbatim command output below, e.g. before and after your change -->
```
before: no message when avisdk<=16.3.5.post1

after: when avisdk<=16.3.5.post1
fatal: [localhost]: FAILED! => {"changed": false, "failed": true, "msg": "Avi python API SDK (avisdk>=16.3.5.post1) is not installed. For more details visit https://github.com/avinetworks/sdk."}
```

Commit:
Added check for avi SDK version as suggested in review.

khaltore commented:
shipit

ericsysmin commented:
shipit

ericsysmin commented:
shipit

gundalow commented:
@abadger Version check as you suggested, how does this seem?

gundalow commented:
I wonder if this code should go into `ansible.module_utils.avi` rather than being duplicated in every module

Have a look at the Amazon `module_utils` for the has boto3 check

grastogi23 commented:
I tried to rearrange the code but I am running into import avi.sdk is throwing an error only in module_utils.avi. I tried other combinations but it works when module is not part of ansible but it does not when I import avi.sdk in module_utils.avi.

Commit:
Fixed documentation based on review.

Commit:
Renamed module_utils.avi to module_utils.avi_ansible_utils
as import of avi.sdk would fail due to name collisions.

Moved the code to check for AVI version into the ansible modules.

grastogi23 commented:
@gundalow @abadger Pl. merge this one before I can add new modules as those PR need to be modified as I am changing the module_utils.avi -> module_utils.avi_ansible_utils to workaround the import of avi.sdk problem inside module_utils.avi.

gundalow commented on lib/ansible/module_utils/avi_ansible_utils.py:
```diff
@@ -28,6 +28,19 @@
 #
 
```
Might be worth putting a comment on the top of this module to state why it's named this way

grastogi23 replied:
sure

Commit:
Updated the module with note about reason for name change.

gundalow commented:
Merged, thanks

You can rebase the other PRs and update to use this functionality.
